### PR TITLE
fix(deps)!: update winit to 0.29

### PR DESCRIPTION
**Pull request by tversteeg**
My editor also applied `cargo fmt`.

It's a breaking change because a winit error is returned and because the winit versions are not compatible.

### CODE REVIEW

**Comment by mkrasnitski** [comment]
There's a lot of extraneous formatting changes in this PR that would better be included in a separate PR. Regardless, this would be great to get merged as I'd like to use `game-loop` in combination with the newest version of `winit`.

File: src/helper.rs [hunk 1]
```diff
@@ -50,44 +68,57 @@ mod helper {
...
+    pub fn game_loop<G, U, R, H, T>(
+        event_loop: EventLoop<T>,
+        window: Arc<Window>,
+        game: G,
+        updates_per_second: u32,
+        max_frame_time: f64,
+        mut update: U,
+        mut render: R,
+        mut handler: H,
+    ) -> Result<(), impl std::error::Error>
```
Comment (mkrasnitski) [inline src/helper.rs]: This should be the concrete `EventLoopError` from `winit` instead of an existential type.

Comment (tversteeg) [reply src/helper.rs]: Thanks for the review!

**Comment by tuzz** [comment]
Thanks @tversteeg, this change is now applied in version `1.1.0` of the crate. I decided not to apply the cargo fmt for now to keep the change clearer in the commit history. Closing now, thanks again.

**Pull request closed without merging**